app/src/main/java/protect/card_locker/DBHelper.java

```diff
@@ -437,7 +447,7 @@ public static boolean updateLoyaltyCard(
             final Date validFrom, final Date expiry, final BigDecimal balance,
             final Currency balanceType, final String cardId, final String barcodeId,
             final CatimaBarcode barcodeType, final Integer headerColor, final int starStatus,
-            final Long lastUsed, final int archiveStatus) {
+            final Long lastUsed, final int archiveStatus ) {
```

Comment:
This whitespace change is unnecessary.

```suggestion
            final Long lastUsed, final int archiveStatus) {
```